bin/oneclick_library_import

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python
 """Update the circulation manager server with new books from the
-OneClick content server."""
+OneClick content server.
+
+If you want to test with sample OneClick data on prod database, run like so:
```

Comment:
Local, QA, or prod.  I had a situation where I wanted to make sure I was able to run a catalog populating script, and save the results to the database, and I didn't want to deal with the entire catalog.  I used the small sample catalog file that I created earlier for a unit test, and was able to troubleshoot my database connection and handling.